### ISSUE TITLE
aptos.getAccountModules()

### ISSUE THREAD

**Issue by RosalinaLuma620** [aptos-labs/aptos-ts-sdk#576]
# 🐛 Bug Description

When I use this SDK to build React Front-end , I invoked the function below .
## How to reproduce

**Code snippet to reproduce**
```typescript
export const getAccountModulesWrapper = async (accountAddress: string) => {
    return await aptos.getAccountModules({
        accountAddress: accountAddress,
        options: {
            limit: 10,
            offset: 1
        }
    })
}
```

**Stack trace / error message**
![image](https://github.com/user-attachments/assets/09fb6cca-8aec-49ac-b45e-f7111e5b94da)

So I checked network records . 

The api request payload is like ( This is how the SDK translates the aptos.getAccountModules() api to do )
**https://api.testnet.aptoslabs.com/v1/accounts/0xf4fd53f1334877ef7aae3241ba87708bf38c618aaa2a22f09e84720fab3328ee/modules?start=11&limit=10**

Notice the query params, is **start** , not **offset**

so I change the link like this to try 
**https://api.testnet.aptoslabs.com/v1/accounts/0xf4fd53f1334877ef7aae3241ba87708bf38c618aaa2a22f09e84720fab3328ee/modules?offset=11&limit=10**

it worked.

## System information

**System details:**
- Typescript SDK Version : Version 5.0.4
- Aptos SDK Version : Version 1.27.1
- Platform : Mac OS M2 

Basically , it's a query params convert issue.

**Comment by gregnazario** [comment]
Yeah, looks like a bug.  Actually amazed it hadn't been hit until now.

Offset is actually a long input, and not just a number to start with, so the PR attached will fix this and the documentat
